charts/pega/README.md

```diff
@@ -232,15 +232,15 @@ Tier name     | Description
 ---           |---
 web           | Interactive, foreground processing nodes that are exposed to the load balancer. Pega recommends that these node use the node classification “WebUser” `nodetype`.
 batch         | Background processing nodes which handle workloads for non-interactive processing. Pega recommends that these node use the node classification “BackgroundProcessing” `nodetype`. These nodes should not be exposed to the load balancer.
-stream (Deprecated)        | For Pega Platform release 8.8 and later, Pega has deprecated the use of embedded "Stream tier" nodes. For new deployments, Pega recommends that you enable an Externalized Kafka configuration under External Services; for existing deployments, nodes that run an embedded deployment of Kafka and are not exposed to the load balancer. Pega requires that these nodes use the node classification “Stream” nodetype.
+stream (Deprecated)        | For Pega Platform release 8.9 and later, pega defaulted to Externalized kafka configuration under external services. for all the new deployments pega won't be using node classification as 'Stream'; for existing deployments, nodes that run an embedded deployment of Kafka and are not exposed to the load balancer. Pega requires that these nodes use the node classification “Stream” nodetype.
```

Comment:
Similarly, for the deprecated 'Stream' node type, the explanation is clear but could be more concise: "From Pega Platform release 8.9 onwards, the use of the 'Stream' node classification is deprecated due to the shift towards an externalized Kafka configuration. For existing deployments where nodes run an embedded Kafka deployment and are not exposed to the load balancer, these nodes should continue using the 'Stream' node type."

Reply:
From Pega Platform release 8.9 onwards-> Pega Platform release '23 onwards,

Reply:
please remember that we reference it like this>>Pega Platform '23
don't say onwards. you have a few choices: 
1 - from version 1 to version 2
2 - For Pega Platform '23 and later

Reply:
from my comments below>>
For Pega Platform '23, the use of the 'Stream' node classification is deprecated; new deployments running version 8.8 and later should not use "Stream" nodes. New deployments connect to a Kafka service that you manage in your organization. For existing deployments using an embedded Kafka deployment which are not exposed to the deployment cluster load balancer, Pega will continue to support the "Stream node classification nodetype.